moto/route53/models.py

```diff
@@ -64,10 +64,26 @@ def __init__(self, health_check_id, caller_reference, health_check_args):
         self.measure_latency = health_check_args.get("measure_latency") or False
         self.inverted = health_check_args.get("inverted") or False
         self.disabled = health_check_args.get("disabled") or False
-        self.enable_sni = health_check_args.get("enable_sni") or False
-        self.children = health_check_args.get("children") or None
+        self.enable_sni = health_check_args.get("enable_sni") or True
         self.caller_reference = caller_reference
 
+    def add_children(self, children):
+        if children and isinstance(children, list):
+            self.children = children
+        elif children and isinstance(children, str):
+            self.children = [children]
+        else:
+            self.children = None
+        return self.children
```

Comment:
The return value is not used anywhere, as far as I can tell, so we can remove this

Reply:
Yep. Missed it. Thanks for pointing out.